src/win/fs.c

```diff
@@ -2053,6 +2105,26 @@ int uv_fs_mkdtemp(uv_loop_t* loop, uv_fs_t* req, const char* tpl,
 }
 
 
+int uv_fs_mkstemp(uv_loop_t* loop, uv_fs_t* req, const char* tpl,
```

Comment:
style: align parameters (I know the windows side is a bit inconsistent, but new code should match the style we want not the one we have :-))